app/units/units.css

```diff
@@ -45,7 +46,7 @@ input:focus, input:first-line {
 .unitSearchInput:invalid{ background-color:transparent; }
 
 .unitSearch .unitSearchInput:focus {
-    border: 2px solid red;
+    outline: none;
```

Comment:
Looks good but would recommend some form of feedback for user.

was thinking something like a thicker line or maybe an accent colour

```css
.unitSearch .unitSearchInput:focus {
    outline: none;
    border: 3px solid #DD432B;
}
```

Reply:
Good catch, for feedback I deleted the border and changed the search bar colour.

Reply:
I like this UI change, thanks!

Reply:
Good Change! Resolving this Convo.